### ISSUE TITLE
How to create mean.bin?

### ISSUE THREAD

**Issue by AlecWong** [dmlc/mxnet#2108]
I have only .rec file, but no mean.bin? So, how to create mean.bin? Help!.... 🎱

**Comment by piiswrong** [comment]
Run the model and it will be computed

**Comment by AlecWong** [comment]
Should I set it manually? Previously, I run the example on cifar-10, and I see that the mean.bin is set manually (mx.io.ImageRecordIter(...., mean_img    = args.data_dir + "mean.bin" ,....).

**Comment by tornadomeet** [comment]
yes,it will generate auto.  you just need give the mean file name.

**Comment by maxkazmsft** [comment]
How is the mean image being computed? Is it computed first time the model is run on the training set, then stored, then subtracted from the images on the test set?
